src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialState = {
+  books: [],
+};
+
+export const BookSlice = createSlice({
+  name: "books",
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books = state.books.filter((book) => book.id !== action.payload);
```

Comment:
- [ ] @Bleu-Yves-Sopoude Great work on your reducer, however, the removeBook action is suppose to check if the book is  not equal to action.id, you can use the example below to ensure the action will work correctly:
```suggestion
      addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },

```